### DISCUSSION TITLE
Domain not working

### DISCUSSION THREAD

**Discussion by sudhamrit** [vercel/community#3058] [Help]
### Summary

 I wanted to reach out regarding an issue I'm currently facing with a domain I purchased from BigRock India. Previously, the domain was working perfectly fine. However, I recently decided to add Cloudflare to enhance the domain's performance and security. As part of this process, I replaced the nameserver from BigRock with Cloudflare's nameservers. Additionally, I added an A record and a CNAME record from Vercel to Cloudflare.

Unfortunately, despite these changes, the domain is still not functioning properly. I'm not sure what could be causing the problem, and I'm seeking assistance to resolve this issue.

I appreciate any help or suggestions you can provide to get the domain back up and running. 

Ref snap:-

![image](https://github.com/vercel/community/assets/131451608/6c57cdaf-3132-4c7b-8056-85f91f63d1af)

### Example

https://www.sudhamrit.in/

### Steps to Reproduce

https://www.sudhamrit.in/

**Comment by smaeda-ks** [comment]
@sudhamrit looks like the redirect loop is caused by your Cloudflare configuration. Your Vercel setup looks fine:

```
# returns 308 from Cloudflare
$ curl -svo /dev/null 'https://www.sudhamrit.in/'

# returns 200 if bypassing Cloudflare
$ curl -svo /dev/null 'https://www.sudhamrit.in/' --connect-to www.sudhamrit.in:443:cname.vercel-dns.com
```

Could you double-check your Cloudflare configuration around http->https redirection?
